serverless.yml

```diff
@@ -268,6 +268,31 @@ functions:
               - name: request.querystring.search_after
               - name: request.header.origin
 
+  # This get_token will get tokens from the elasticsearch index
+  es_get_token_handler:
```

Comment:
I know this project style is to use snake case for lambda names but AWS translates to camel case replacing each underscore with `Underscore` for some internal names (e.g. policy statement name) this makes the name longer and easier to hit the limit, failing the deploy.

I think we should create new lambdas with camel case to avoid these issues.